src/kata6.joinNames.js

```diff
@@ -1,3 +1,9 @@
-const joinNames = namesObj => {};
+const joinNames = (namesObj) => {
```

Comment:
Brackets not needed here
```suggestion
const joinNames = namesObj => {
```